pgrx-sql-entity-graph/Cargo.toml

```diff
@@ -43,3 +43,4 @@ syntect = { version = "5.2.0", default-features = false, features = ["default-fa
 [lints.clippy]
 assigning-clones = "allow" # wrong diagnosis and wrong suggestions
 too-many-arguments = "allow" # I argue with myself all the time
+collapsible-if = "allow" # nuisance in the way of updating editions
```

Comment:
evaluating the logic of these changes should be done with more attention for them specifically than I wish to spare at the moment.